modules/ndu/src/translations/fr.json

```diff
@@ -23,13 +23,15 @@
       }
     },
     "workflowCalled": "Le workflow est déclenché par un autre workflow",
-    "userAlreadySpoke": "L’utilisateur a déjà parlé avec l’agent conversationnel",
-    "userAuthenticated": "L’utilisateur est authentifié",
-    "userComingFromTopic": "L’utilisateur vient de {topicName}",
-    "userEndedWorkflow": "L’utilisateur a mis fin à un flux de travail",
-    "userNotInWorkflowOrBlock": "L’utilisateur n’est pas dans un flux de travail ou un bloc",
-    "userSpeaks": "L’utilisateur parle {language}",
-    "userTyped": "L’utilisateur a tapé quelque chose de spécifique",
-    "userUsingChannel": "L’utilisateur utilise {channelName}"
+    "userAlreadySpoke": "L'utilisateur a déjà parlé avec l'agent conversationnel",
```

Comment:
I suggest we use only "agent" instead of "agent conversationnel" ... Seems so serious & tedious

Reply:
I just used find/replace to fix french quotemarks to the correct quotemarks. Didn't look at the text at all